tools/mkpkg/update_binary-addons

```diff
@@ -9,104 +9,23 @@ if [ -z "$1" ]; then
   exit 0
 fi
 
-ROOT=$(cd $(dirname $0)/../.. && pwd)
+# the following environment variables can be set to "yes" to enable optional features:
+# KEEP_GIT_DIRS: don't delete cloned git directories after update check
+# BUMP_PKG_REV: bump PKG_REV if PKG_VERSION has changed
 
-KODI_BRANCH="$1"
-KODI_DIR="kodi-${KODI_BRANCH}.git"
-
-git_clone() {
-  # git_clone https://repo.url branch ./target_dir [githash]
-  echo "[mkpkg] Checking out $1 ..."
-  if [ ! -d "$3" ]; then
-    git clone "$1" "$3"
-    # Try to switch to specified branch if it exist, if not then use default branch
-    if [ -n "$2" ]; then
-      cd "$3"
-      git checkout $2 >/dev/null 2>/dev/null
-      cd ..
-    fi
-  else
-    if [ -d "$3" ] ; then
-      cd "$3"
-      git checkout $2 >/dev/null 2>/dev/null
-      git pull
-      cd ..
-    fi
-  fi
-
-  if [ ! -z "$4" ] ; then
-    cd "$3"
-    git fetch >/dev/null 2>/dev/null
-    git branch -D $4 >/dev/null 2>/dev/null
-    git checkout $4 >/dev/null 2>/dev/null
-    git checkout -b ref-$4 >/dev/null 2>/dev/null
-    cd ..
-  fi
-}
-
-resolve_hash() {
-  if [ -d "$1" ] ; then
-    cd "$1"
-    git rev-parse $2 2>/dev/null
-  fi
-}
-
-# Get url in git:// notation for a package.mk, assuming it is a github.com url
-# Return 1 if not a github domain
-geturl() {
-  local addon="$1"
-  local domain owner repo PKG_URL
-
-  PKG_URL="$(get_pkg_var ${addon} PKG_URL)"
-
-  domain="$(echo "${PKG_URL}" | cut -d/ -f3)"
-  [ "${domain}" = "github.com" ] || return 1
-
-  owner="$(echo "${PKG_URL}" | cut -d/ -f4)"
-  repo="$(echo "${PKG_URL}" | cut -d/ -f5)"
-
-  echo "git://${domain}/${owner}/${repo}.git"
-  return 0
-}
-
-# For the specified addon, verify that the package url
-# matches the url retrieved from Kodi
-validate_pkg_url() {
-  local addon="$1" url1="$2"
-  local domain owner repo url2
+MY_DIR=$(dirname $0)
+ROOT=$(cd "${MY_DIR}"/../.. && pwd)
```

Comment:
```
MY_DIR="$(dirname $0)"
ROOT="$(cd "${MY_DIR}"/../.. && pwd)"
```
?

Reply:
thanks, yes, and $0 needs to be quoted as well